### ISSUE TITLE
bluetooth-battery@zamszowy: error adding applet

### ISSUE THREAD

**Issue by G4Zz0L1** [linuxmint/cinnamon-spices-applets#4338]
```
 * Applet version/Build date (to be found in "About.." or in "metadata.json"): "last-edited": 1652814982
 * Cinnamon version (cinnamon --version): Cinnamon 5.2.7
 * Distribution - (Mint 17.2, Arch, Fedora 25, etc...): Arch Linux
 * Graphics hardware *and* driver used: AMD Radeon RX 6900 XT with amdgpu 4.6 Mesa 22.0.3
 * Snippet or attachment of ~/.cinnamon/glass.log
 * Snippet or attachment of ~/.xsession-errors
 ```

**Notify author of applet**
There is no author in the CODEOWNERS file

**Issue**
When I add this applet it shows those lines on the looking glass log section
```
error t=2022-05-18T11:00:39Z GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: Method Refresh is not implemented on interface org.freedesktop.UPower.Device
error t=2022-05-18T11:00:39Z 
[bluetooth-battery@zamszowy]: Applet bluetooth-battery@zamszowy: Could not create applet object.
[bluetooth-battery@zamszowy]: Error importing applet.js from bluetooth-battery@zamszowy
```

On the .xsession-errors
```
(cinnamon:7128): Cjs-CRITICAL **: 10:58:03.358: JS ERROR: Gio.DBusError: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: Method Refresh is not implemented on interface org.freedesktop.UPower.Device
setup_dbus@/home/user/.local/share/cinnamon/applets/bluetooth-battery@zamszowy/applet.js:191:17
setup@/home/user/.local/share/cinnamon/applets/bluetooth-battery@zamszowy/applet.js:94:14
```

**Comment by tnowak12** [comment]
I've opened pull request with tfix that addresses this #4341

**Comment by zamszowy** [comment]
I've opened pull request with the fix that addresses this #4341
